app/models/user.rb

```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
```

Comment:
According to the requirements:

![image](https://user-images.githubusercontent.com/57421823/169996098-2682fd17-9d61-4c1c-a00a-0de75f1f0961.png)

Therefore:

- [ ] Please, make sure a new user can be registered through the "Sign up" form.

## Explanation

- Right now, it is impossible to register a new user or log in.  Even in the console.
- This happens when trying:

![image](https://user-images.githubusercontent.com/57421823/169996413-a6e72c53-a7ca-4c08-8e85-5cac2ab4da42.png)

- Possible reason from devise's instructions:

![image](https://user-images.githubusercontent.com/57421823/170000646-1c3dac37-452c-45a9-b055-574628de368c.png)

You copied the command as-is, instead of replacing the 'MODEL' word with 'user' which is your class that creates users.  The result is that instead of updating your 'users' table, devise created a 'MODEL' table, which shouldn't exist.  Look:

![image](https://user-images.githubusercontent.com/57421823/170001590-a004e9a4-4567-42fc-8aa2-044573b344ff.png)

All these columns should have been added to your 'users' table by the generator command.

Please, kindly correct this bug so devise is correctly installed.